### ISSUE TITLE
Update Loadouts

### ISSUE THREAD

**Issue by jerkinmerkin** [Spearhead-Gaming-Milsim/OP_Red_November#1]
Update loadouts to give SOF the worst gear possible because I hate them.

**Comment by jerkinmerkin** [comment]
But seriously, need to update loadouts just a tad to reflect lessons learned from OP Queens Horses.

**Comment by FoxKilo** [comment]
I hate you too. Easiest thing to do would be to grab all the COs and have a development session where errybody can negotiate what kits/vehicles/weapon systems they'll have available to them. Everyone then knows what capability each other has and can address issues like the one Cavalry had (i.e no weapons teams, therefore make kit available to Cav). Capisci? Da

**Comment by jerkinmerkin** [comment]
I plan on opening up separate issue threads for each unit as I put together the loadouts.  There I'll post the config.cpps or .sqfs so each group can review and give feedback.  Of course I reserve the right as a mission maker reject feedback and requests that don't fit the scenario but at least it opens up a dialog.

**Comment by jerkinmerkin** [comment]
I'll also include a description of each unit in the scenario so unit COs can better understand who they're portraying and what I envision as a mission maker.

**Comment by FoxKilo** [comment]
Yeah that sounds fair, I mean that's how it was always meant to be in the first place but everyone got complacent in not helping the mission maker, myself included at the time. You going to be switching to SQFs on this mission too or from Hand of God?

**Comment by jerkinmerkin** [comment]
This one will be using configs since they're already 98% finished.  Hand of God will use .sqfs.

**Comment by coachwn** [comment]
They already have that option and each CO/Team leader has been directed to build their preferred loadouts in the separate repositories that are linked in this github. Those will be the base loadouts and mission makers can adjust as needed based on the scenario. I want to leave it up to the mission builder to communicate with the various CO's regarding the loads on a per mission basis.

**Comment by jerkinmerkin** [comment]
Yes, I'll use those as base reference for my loadouts.  Then post mission specific loadouts to this repo for discussion.

**Comment by jerkinmerkin** [comment]
WTF should I do with joint fires?

**Comment by coachwn** [comment]
Been away for the most part today...For Red November I would say attach Joint Fires can be their own entity like the rest of the groups in this aspect. It is difficult to give them a true slot since they will not really have a mission in Red November. I will talk with Bayley unless you get to him first @jerkinmerkin; because I just don't know how they would slot into the proposed mission.

**Comment by FoxKilo** [comment]
Since there are only a couple of them on the roster you could post them as;

- Gunner team (Crew served)
- Mortar team
- Logistics (asset recovery, enemy & friendly)(air logi, fuel/rearm)(medevac drivers) and so forth.
- Infantry supplements
